pkg/cluster/cluster.go

```diff
@@ -169,10 +169,11 @@ func New(ctx context.Context, log *logrus.Entry, _env env.Interface, db database
 	if err != nil {
 		return nil, err
 	}
-	fpspID, err := token.GetObjectId(t.Token)
+	tokenClaims, err := token.ExtractClaims(t.Token)
 	if err != nil {
 		return nil, err
 	}
+	fpspID := tokenClaims.ObjectId
```

Comment:
Wondering if we should name the parameter to specifically call it out as the instantiation in the customer's tenant, not our own.